docs/automations/integrations/copilot/flag-copilot-pr/README.md

```diff
@@ -93,4 +93,26 @@ Automatically apply labels to PRs that are assisted by GitHub Copilot. You can a
         [:octicons-download-24: Download this example as a CM file.](/downloads/automation-library/integrations/copilot/label_copilot_by_tag.cm){ .md-button }
         </span>
         </div>
+
+=== "Label by Copilot code comment"
```

Comment:
Currently, it's only in Experimental mode and may be changed in the future. 
I think it's better to mention it.